R/cookies.R

```diff
@@ -195,14 +193,14 @@ appendCookieHeaders <- function(requestURL, headers) {
   cookies <- get(host, envir = .cookieStore)
 
   # If any cookies are expired, remove them from the cookie store
-  if (any(cookies$expires < as.integer(Sys.time()))) {
+  if (isTRUE(any(cookies$expires < as.integer(Sys.time())))) {
```

Comment:
@karawoo - Took this suggestion from you even though we are now avoiding the NA values.